src/Uno.UI/Controls/BindableSearchBar.iOS.cs

```diff
@@ -32,7 +32,7 @@ public partial class BindableSearchBar : UISearchBar, DependencyObject, System.C
 		private static readonly TimeSpan _defaultTextChangedMinDelay = TimeSpan.FromMilliseconds(250);
 		private const bool _defaultIsAutoLostFocusEnabled = true;
 
-		private readonly SerialDisposable _textChangedSubscription = new SerialDisposable();
+		//private readonly SerialDisposable _textChangedSubscription = new SerialDisposable();
```

Comment:
@jeromelaban I kept it since there is a commented reference to it:

https://github.com/unoplatform/uno/blob/746d6867c3d21bdb355d48d8fb70ccc2cb9be879/src/Uno.UI/Controls/BindableSearchBar.iOS.cs#L187

Same applies to everything that I commented out and not removed.

My intent was to make it easy if the commented code will be uncommented later.

I don't mind removing though. Let me know if you still think I should remove.